sec_certs/sample/common_criteria.py

```diff
@@ -684,6 +696,41 @@ def compute_heuristics_cert_id(self, all_cert_ids: Set[str]):
         self.heuristics.cert_id = self.pdf_data.cert_id
         self.normalize_cert_id(all_cert_ids)
 
+    def _get_dependency_cves(self, dset: "CCDataset", dependency_type: DependencyCVEType) -> Optional[Set[str]]:
+        dependency_type_dict = {
+            "direct": self.heuristics.report_references.directly_referenced_by,
+            "indirect": self.heuristics.report_references.indirectly_referenced_by,
```

Comment:
**Chore**: `"direct"` and `"indirect"` should be probably replaced with the `Enum`, right?